lib/logstash/outputs/elasticsearch.rb

```diff
@@ -608,6 +612,12 @@ def setup_ecs_compatibility_related_defaults
     @template_name ||= default_template_name
   end
 
+  def setup_template_manager_defaults(data_stream_enabled)
+    if @manage_template.nil? # not set
+      @manage_template = !data_stream_enabled
```

Comment:
```suggestion
+    if original_params["manage_template"].nil? && data_stream_enabled
+      logger.debug("Disabling template management since data streams are enabled")
+      @manage_template = false
```